src/i18n/de.json

```
@@ -537,7 +537,8 @@
         "github-error": "Beim Absenden gab es ein Problem mit GitHub. Dein Vorschlag wurde nicht abgeschickt. Versuch es doch noch einmal.",
         "view-on-github": "Hier kannst Du den Fortschritt auf GitHub verfolgen",
         "ok": "Ok",
-        "similarly-named-companies": "Wir haben schon diese Firmen mit ähnlichem Namen:"
+        "similarly-named-companies": "Wir haben schon diese Firmen mit ähnlichem Namen:",
+        "no-change": "Es gibt keine Änderungen die vorgeschlagen werden können."
```

I think I would prefer something like:

```suggestion
        "no-change": "Du hast nichts geändert."
```

But if you want to keep this:

```suggestion
        "no-change": "Es gibt keine Änderungen, die vorgeschlagen werden können."
```